### PR TITLE
Basics of machine learning - fix

### PR DESCRIPTION
**Pull request by hannesdatta**
@K-sohooli, I have now finally gotten around reviewing your original [PR](https://github.com/tilburgsciencehub/website/pull/986).

The reason it took so long is that there were a few infrastructural issues that caused a lot of problems to check and integrate your material. I have now gotten to a point that the material has the "right place" on our website. So you probably don't have to worry about it.

However, the content is not yet how I would like it. I will give you a bit of specific instructions here and hope you can revise the content of this PR.

- the content right now is overwhelming; it's a lot of text, a lot of formulas, and it will be hard for our users to process. please think a bit about what the purpose is of this article (don't call it bb anymore - we now write 'articles'): raise awareness? give a practical example? or reproduce a text book?
- I think a good repitch of this would be to (a) explain the basic concepts (in simple language!), and show some practical implementation on "why it's important to learn it".
- in other words, I ask you to simplify and streamline the language
- cut down text
- integrate practical, relevant examples
- also, ensure you tie both articles together by linking to them.

Think about: what would YOU like to read about it?

I hope you can retry/update.

Thank you and be in touch if you need any feedback!

### CODE REVIEW

**Comment by hannesdatta** [comment]
BTW: Forgot to say: I do like that we will cover these type of issues!!! I do feel we must have content like this.

**Comment by lachlandeer** [comment]
new version of this work is in #1095

**Comment by K-sohooli** [comment]
@lachlandeer <https://github.com/lachlandeer> , all the images relevant to
this article are included. Regarding the source of images,
"costfunctionplot_2" and "2_dim_costfunction" are taken from a course by
Andrew Ng on the Coursera platform. and I created four other images in
Microsoft Word and draw.io.
